test/unit/queue/src/QueueTest.cpp

```diff
@@ -112,6 +112,8 @@ TEMPLATE_LIST_TEST_CASE(
         alpaka::wait(f.m_queue);
     }
 
+    // without this call thread sanitizer checks will show errors
+    alpaka::empty(f.m_queue);
```

Comment:
Removing this line will bring back sanitizer errors. Currently, I do not know why.
Currently we call `m_callbackThread.empty()` within `alpaka::empty(f.m_queue)`. Errors are silent too if we call `m_callbackThread.empty()` for CUDA/HIP queues here instead of adding the call to empty but not all queue types have the member `m_callbackThread`.

Never the less this line is only to show how to silent the errors and should be a discussion base to understand what going on.